.github/dependabot.yaml

```diff
@@ -1,10 +0,0 @@
-version: 2
-updates:
-  - package-ecosystem: "gitsubmodule"
-    directory: "/"
-    schedule:
-      interval: "daily"
-  - package-ecosystem: "github-actions"
-    directory: "/"
-    schedule:
-      interval: "monthly"
```

Comment:
Do we really want to remove this part? IIUC, it's there to update the external Github Actions we use in our workflows, like `actions/checkout@v2` and `actions/upload-artifact@v2`?